src/abstract-interpretation/data-frame/absint-info.ts

```diff
@@ -1,36 +1,34 @@
 import type { NodeId } from '../../r-bridge/lang-4.x/ast/model/processing/node-id';
 import type { DataFrameDomain } from './domain';
-import type { DataFrameOperationName } from './semantics';
+import type { DataFrameOperationName } from './expression-semantics';
 
 export interface DataFrameOperation {
 	operation: DataFrameOperationName,
 	operand:   NodeId | undefined,
-	arguments: (NodeId | undefined)[]
+	arguments: (NodeId | undefined)[],
+	modify?:   boolean
 }
 
-interface DataFrameStatementInfo {
-	type:   'statement',
-	domain: Map<NodeId, DataFrameDomain>
+interface DataFrameInfo {
+	type:    string;
```

Comment:
We can discuss this on friday, for now I am fine with leaving the super-type unbounded.